pipenv_devcheck/check_fns.py

```diff
@@ -1,70 +1,78 @@
 import operator
 
 
-def check_equality(setup_version, pipfile_op, pipfile_version):
+def check_equality(left_version, right_op, right_version, **_):
```

Comment:
This isn't blocking, but I think it was a little clearer/easier to understand when the variables were setup and pipfile version, especially if you were trying to debug something.

Reply:
I agree with you 100% - however, there are scenarios where you need the dependencies from the Pipfile to be the on the left side of the equation, which was previously called `setup_version` and `setup_op`. As soon as I saw that I was passing Pipfile deps to setup variables, I realized that a change would be needed whether I liked it or not. Even if calling them 'setup_*` and `pipfile_*` is no longer accurate, that doesn't mean I'm sold on these - I'd happily hear out anything that you think would be a better name.